src/index.js

```diff
@@ -37,6 +37,7 @@ const makeSchema = (schemaOptions, schemaExtension) => {
     output: outputSchema,
     plugins: pluginsSchema,
     profile: Joi.boolean(),
+    progress: Joi.boolean(),
```

Comment:
Its the node equivalent of the cli flag `--progress`
https://webpack.github.io/docs/cli.html#progress